lewis_emulators/wbvalve/interfaces/stream_interface.py

```diff
@@ -0,0 +1,47 @@
+from lewis.adapters.stream import StreamInterface, Cmd
+from lewis.core.logging import has_log
+
+from lewis_emulators.utils.command_builder import CmdBuilder
+from lewis_emulators.utils.replies import conditional_reply
+
+if_connected = conditional_reply("connected")
+
+@has_log
+class WbvalveStreamInterface(StreamInterface):
+
+    in_terminator = '\n'
+    out_terminator = '\n'
```

Comment:
Terminator here doesn't match the one you have set in the support module. I think this one is correct.